docs/x509/verification.rst

```diff
@@ -133,6 +133,10 @@ the root of trust:
 
     .. versionadded:: 43.0.0
 
+    .. versionchanged:: 44.0.0
+        ``verification_time`` and ``max_chain_depth`` were replaced by the 
+        ``policy`` property that provides access to these values.
+
```

Comment:
We may want to keep these around as proxies, but this isn't terrible important.

Reply:
I would add that at a later time if it's deemed required. The API is declared as being unstable in the docs, but I understand that we shouldn't go out of our way to introduce breaking changes 😄 Let's keep this conversation unresolved so once we get back to this PR we can look into this again.